src/book/mdx.ts

```diff
@@ -1,42 +1,25 @@
-import fs from 'fs';
-import { join } from 'path/posix';
+import { Plugin } from 'esbuild';
 
-/**
- * Compiles MDX file into a React component.
- * https://mdxjs.com
- * https://www.npmjs.com/package/@mdx-js/mdx
- * @param path
- * @param dir
- */
-export async function compileMdx (path: string, dir: string): Promise<string | undefined> {
-  if (fs.existsSync(path)) {
-    const content = fs.readFileSync(path, 'utf-8');
+export async function createMdxPlugin (): Promise<Plugin> {
+  // TODO(burdon): Figure out how to use EMS with CJS.
+  const { default: mdx } = await eval('import("@mdx-js/esbuild")');
```

Comment:
why do these need to be evaled inside the plugin?

Reply:
They only support ESM and the "bin" file installed by the package starts with `require`. Couldn't figure out a better approach.